bower.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "purescript-routing",
+  "version": "0.1.0",
```

Comment:
Version fields in `bower.json` are redundant at the moment as it only uses git tags to resolve released versions anyway.